k2/csrc/log.h

```diff
@@ -93,6 +96,11 @@ class Logger {
       __assert_fail("Some bad things happened", filename_, line_num_,
                     func_name_);
 #else
+      std::string stack_trace = GetStackTrace();
```

Comment:
Can you guys build successfully with the latest master branch? it seems to me `GetStackTrace()` is defined in `log.cu`, but `log.h` is referenced in those /host code, so I will get error 
```
bin/ld: CMakeFiles/weights_test.dir/weights_test.cc.o: in function `k2::internal::Logger::~Logger()':
/ceph-hw/k2/k2/csrc/log.h:99: undefined reference to `k2::internal::GetStackTrace()'

```
I suppose @csukuangfj has definitely built successfully before committing code?